src/components/Article/Article.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Article = ({ title, text, date }) => (
+  <>
+    <h1>
+      {title}
+    </h1>
+
+    <p>
+      {text}
+    </p>
+
+    <span>
+      {date}
+    </span>
```

Comment:
It looks more compact and readable. 
```suggestion
    <h1>{title}</h1>
    <p>{text}</p>
    <span>{date}</span>
```